.github/workflows/performance.yml

```diff
@@ -49,3 +49,37 @@ jobs:
         env:
           SWIFT_SUFFIX: 'Swift'
           SAMPLE: Performance
+      - name: Build and Test SwiftUI iOS
+        run: |
+          set -io pipefail
+          cd performance/SwiftUIPerformanceQuickstart/PerformanceExample
+          bundle exec pod install --repo-update
+          xcodebuild \
```

Comment:
The test is for PRs with secrets access only. We might only run build for forks.

Reply:
Should I revert to checking for secrets?

Reply:
As discussed in the meeting, we can revert it to the previous if-else version, but `set output` to let the `have_secrets` be public to other steps.

Reply:
Because these scripts will later be consolidated, I instead used `check_secrets.sh` to achieve this.